kalite/updates/static/js/updates/bundle_modules/update_videos.js

```diff
@@ -178,7 +178,7 @@ $(function() {
                         // Best to assume offline, as online check returns much faster than offline check.
                         if(server_is_online){
                             $(".enable-when-server-online").removeAttr("disabled");
-                            base.updatesStart("videodownload", 5000, video_callbacks);
+                            // base.updatesStart("videodownload", 5000, video_callbacks); no idea why run this here?
```

Comment:
@66eli77 we run this here in case there are any currently running download jobs going, so we start polling as soon as you load the page.